doc/releases/nautilus.rst

```diff
@@ -0,0 +1,506 @@
+v14.1.0 Nautilus
+================
+
+.. note: These are draft notes for the first Nautilus release.
+
+This is the first stable release of Nautilus.
+
+Major Changes from Mimic
+------------------------
+
+- *Dashboard*:
+
+
+- *RADOS*:
+
+  * The new *msgr2* wire protocol brings support for encryption on the wire.
+  * The number of placement groups (PGs) per pool can now be decreased
+    at any time, and the cluster can automatically tune the PG count
+    based on cluster utilization or administrator hints.
+  * Physical storage devices consumed by OSD and Monitor daemons are
+    now tracked by the cluster along with health metrics (i.e.,
+    SMART), and the cluster can apply a pre-trained prediction model
+    or a cloud-based prediction service to warn about expected
+    HDD or SSD failures.
+
+- *RGW*:
```

Comment:
S3 lifecycle transition for storage class tiering
STS/AssumeRoleWithWebIdentity (auth federation using OAuth2 and OpenID::connect
User Policy and Bucket Policy enhancements
Bucket move (rename buckets leaving data at rest)
archive zone
improved Swift/S3 interoperability
Initial Publish-Subscribe notification support
Beast Frontend as default

Reply:
S3 append object extension

Reply:
Initial dmclock QOS implementation (minimal set of priority classes)

Reply:
OPA authn delegation prototype